bindings/go/src/fdb/transaction.go

```diff
@@ -337,6 +337,12 @@ func (t Transaction) Clear(key KeyConvertible) {
 // ClearRange removes all keys k such that begin <= k < end, and their
 // associated values. ClearRange returns immediately, having modified the
 // snapshot of the database represented by the transaction.
+// Range clears are efficient with FoundationDB -- clearing large amounts of data
+// will be fast. However, this will not immediately free up disk -
+// data for the deleted range is cleaned up in the background.
+// This will not affect the database until commit is called.
+// Actual amount of stored data for the specified range does not count against
```

Comment:
```suggestion
// The actual amount of stored data for the specified range does not count against
```
And then similarly for the other comments.

Reply:
copied the blurb provided by AJ - which makes this change unnecessary.